packages/flutter_sprites/lib/src/action.dart

```diff
@@ -1,5 +1,10 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
 part of flutter_sprites;
 
+/// Callback used by the [ActionCallFunction].
```

Comment:
for typedefs, we use the style "Signature for ..." e.g. "Signature for callbacks used by [ActionCallFunction] objects."